packages/angular/cli/lib/main.ts

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
`cli/lib/index.ts` can be renamed to `main.ts` instead, no need for an extra file.

Reply:
Ok, but than it will be exported with a default export. That is OK?

Reply:
I think the default export is ok, yes.